src/components/tabs/js/tabsController.js

```diff
@@ -261,7 +261,7 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
     ctrl.selectedIndex     = getNearestSafeIndex(newValue);
     ctrl.lastSelectedIndex = oldValue;
     ctrl.updateInkBarStyles();
-    updateHeightFromContent();
+    if(!ctrl.stretchHeight) updateHeightFromContent();
```

Comment:
Add space for consistency please: `if(` -> `if (`

Reply:
@Splaktar addressed.